### ISSUE TITLE
Picking output directory dialogue closes instantly

### ISSUE THREAD

**Issue by Aleosiss** [gorilla-devs/ferium#351]
### Description

When attempting to set a mod output directory, the dialogue instantly closes and the I am unable to enter a path.

https://github.com/gorilla-devs/ferium/assets/25385833/4a78fcd9-4e9c-40c3-9311-7e277a3db871

### Steps to reproduce

From the `history` output on a fresh install of Ubuntu on WSL2.0
    1. `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
    2. `source "$HOME/.cargo/env"`
    3. `sudo apt-get update`
    4. `sudo apt install build-essential`
    5.  `cargo install ferium`
    6.  `ferium modpack add kosmolot-unofficial-masa-mods`

### Operating System

Linux

### Are you using the latest version of ferium?

- [X] I am using the latest version of ferium

### Additional Information

Installing on WSL2

**Comment by theRookieCoder** [comment]
I would recommend using the no-gui version (`cargo install ferium --no-default-features`) for now while I figure this out.

**Comment by ok-coder1** [comment]
@theRookieCoder, you could add a option in `ferium` for typing the path in.

**Comment by theRookieCoder** [comment]
> @theRookieCoder, you could add a option in `ferium` for typing the path in.

There is, all commands that have any directory argument have command line flags for it.

**Comment by ok-coder1** [comment]
So shouldn't this be closed?

**Comment by theRookieCoder** [comment]
No, the GUI file dialogue still doesn't work via WSL to my knowledge?

**Comment by ok-coder1** [comment]
How does the dialog box work? It's supposed to be using the Windows dialogue box as AFAIK, WSL doesn't have the Linux file selection dialog right?

**Comment by theRookieCoder** [comment]
I use this library called [rfd](https://docs.rs/rfd), on linux (including anything running inside WSL) it will communicate using the xdg desktop portal to use the system's default file picker.
